CrisisTextLine/Elmer
# Added code to retry users whose Canvas updates fail the first time.

#### What's this PR do?
Checks after the attempted Canvas update to a given user's grade for "Schedule Your Shifts" to see if there was any problem in updating the grade. If there was, it resets the user's When I Work notes so that the CronJob will retry a Canvas update for the user the next time it runs.
#### Where should the reviewer start?
updateCanvas.js, NotifyMoreShifts.js
#### How should this be manually tested?
Npm test and:
1. Create a fake user in When I Work and assign him to the test location.
2. Give your fake user a shift.
3. Add the requisite GLOBAL.KEYS and GLOBAL.CONFIG declarations to NotifyMoreShifts.js and WiWCCApi.js. Change the location ID in NotifyMoreShifts.js (line 26) to "test."
4. Change "retryUser" on line 133 of NotifyMoreShifts.js to "true."
5. node jobs/scheduling/NotifyMoreShifts.js.
6. The code will initially set the user's notes to "firstShiftNotification:true," but the new retry script, which we've ensured in (4) above will say that the Canvas update failed, should change that to "firstShiftNotification:undefined".
7. Check the user's notes in WiW to confirm (6).

#### Any background context you want to provide?
#### What are the relevant tickets?
[INT-322
](https://admin.crisistextline.org/jira/browse/INT-322)#### Questions:
